testbed/stabilitytests/.gitignore

```diff
@@ -0,0 +1,2 @@
+results/*
+!results/.gitkeep
```

Comment:
This file will not be needed once this PR is merged https://github.com/open-telemetry/opentelemetry-collector/pull/1116